include/AutomationTrack.h

```diff
@@ -31,7 +31,7 @@
 #include "TrackView.h"
 
 
-class AutomationTrack : public Track
+class LMMS_EXPORT AutomationTrack : public Track
```

Comment:
Why does this PR add `LMMS_EXPORT` for multiple classes?

Reply:
Since this PR changes the object library to a shared library, this is required on Windows.